onnxruntime/core/optimizer/common_subexpression_elimination.cc

```diff
@@ -412,7 +529,20 @@ Status CommonSubexpressionElimination::ApplyImpl(Graph& graph, bool& modified, i
          output_idx < end; ++output_idx) {
       const NodeArg* output_def = node->OutputDefs()[output_idx];
 
-      const EquivalenceClass* equivalence_class = equivalence_classes.at(output_def);
+      const EquivalenceClass* equivalence_class = nullptr;
+      
+      if(graph.IsInitializedTensor(output_def->Name())) {
```

Comment:
Can this ever happen? My understanding is that `output_def` here is an output of the `node`, so it cannot be an initializer or a graph input.

Reply:
I missed it completely - will try to wrap it in the next set of changes.